protocol/daemons/types/health_checkable.go

```diff
@@ -119,12 +128,15 @@ func (h *timeBoundedHealthCheckable) HealthCheck() error {
 
 	// If the last successful update was more than 5 minutes ago, report the specific error.
 	if h.timeProvider.Now().Sub(h.lastSuccessfulUpdate) > MaxAcceptableUpdateDelay {
-		return fmt.Errorf(
-			"last successful update occurred at %v, which is more than %v ago. Last failure occurred at %v with error '%w'",
-			h.lastSuccessfulUpdate,
-			MaxAcceptableUpdateDelay,
-			h.lastFailedUpdate.Timestamp(),
-			h.lastFailedUpdate.Error(),
+		h.logger.Error(
```

Comment:
downgrade error to a log message.

Reply:
To confirm, we want this to take effect for all daemons right? (prices + liquidation + bridge)? 

I think I am aligned but wanted to call this will affect all daemons

Reply:
Yes!

Reply:
Wait, if we change this from return an error to just logging an error, how would we enforce `[High] Panic on the protocol side if daemon is unhealthy for X minutes (aka grace period, X = 5 to 15 min TBD)`?

Wouldn't this change mean that we never report unhealthy back to the protocol side? Consider the scenario where the last success time was Y and the last failure time was Y-1 and Y was a long time ago and the daemon was not able to make any new successful or failure loop? Is it guaranteed that the loops would always report success or failure within a certain strict time window?

Reply:
No, not gauranteed. We discussed this and considered the risk acceptable. The long-term fix is to add timeouts to gRPC queries, which is the most likely source of loop non-termination... But in practice we haven't seen the daemon hang on a gRPC call except during initialization.